src/translations/en.json

```diff
@@ -1503,12 +1503,12 @@
               "themes": "Themes"
             },
             "server_management": {
-              "heading": "Server management",
-              "introduction": "Control your Home Assistant.",
-              "restart": "Restart",
-              "confirm_restart": "Are you sure you want to restart Home Assistant?",
+              "heading": "Home Assistant Core",
+              "introduction": "Restarting Home Assistant Core will stop your dashboard and automations. After the reboot, each configuration will be reloaded.",
+              "restart": "Restart Core",
+              "confirm_restart": "Are you sure you want to restart Home Assistant Core?",
               "stop": "Stop",
-              "confirm_stop": "Are you sure you want to stop Home Assistant?"
+              "confirm_stop": "Are you sure you want to stop Home Assistant Core?"
```

Comment:
to a user, its just home assistant. they don't necessarily need to know HA Core vs HA